trustpoint/pki/initializer/issuing_ca/__init__.py

```diff
@@ -1,17 +1,13 @@
-from .. import Initializer
-from .. import InitializerError
-
-from .base import IssuingCaInitializer
-from .base import IssuingCaInitializerError
+"""Initialization module for different types of issuing CAs."""
 
+from .. import Initializer, InitializerError
```

Comment:
Is this a relative import? Never seen a double dot ^^.
If so please use absolute path

Reply:
I think so :D either similar code was already there or `ruff` formatted it this way. Should be no problem to change it to absolute path though. In any case, we do have quite a few `from .models import` and `from . import` instances in the codebase, but those should be fine as they only reference modules in the same folder.